### PR TITLE
Photo url copies to clipboard when taking a picture now

### PR DESCRIPTION
**Pull request by lenzh**

### CODE REVIEW

File: client/nui/app.js [hunk 1]
```diff
@@ -42,6 +41,15 @@ function close() {
...
 
+const copyToClipboard = (str) => {
+  const el = document.createElement("textarea");
+  el.value = str.replace(/[" ]/g, "");
+  document.body.appendChild(el);
+  el.select();
+  document.execCommand("copy");
+  document.body.removeChild(el);
+};
+
```
Comment (xFutte) [inline client/nui/app.js:52]: Not a very good implementation, including that `document.execCommand()` has been deprecated. Have a look at https://developer.mozilla.org/en-US/docs/Web/API/Navigator/clipboard instead.

**Review by xFutte — CHANGES_REQUESTED** [review]

**Comment by xFutte** [comment]
Looks good to me.

**Comment by xFutte** [comment]
@MonkeyWhisper Why was this merged?

**Comment by MonkeyWhisper** [comment]
> @MonkeyWhisper Why was this merged?

Shoot, I didn't see your comment above!

**Pull request merged**